whois-nrtm4/src/main/java/net/ripe/db/nrtm4/DeltaChange.java

```diff
@@ -0,0 +1,57 @@
+package net.ripe.db.nrtm4;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+
+
+public class DeltaChange {
+
+    @JsonSerialize(using = EnumToStringSerializer.class)
+    enum Action {
+        DELETE,
+        ADD_MODIFY;
+
+        @Override
+        public String toString() {
```

Comment:
I guess you can directly call  @JsonValue on a getter method instead of creating a custom serializer?